src/index.html

```diff
@@ -22,6 +22,404 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div class="moyo_catalog"></div>
+    <header class="header">
+      <a
+        href="#"
+        class="moyo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_item__link is-active"
+            >
+              Apple
+            </a>
```

Comment:
The href attribute is set to '#', which might not be functional. Consider providing a valid URL or ensuring this is intentional for navigation purposes.